http-common/src/main/java/com/wultra/core/http/common/headers/UserAgent.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2023 Wultra s.r.o.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.wultra.core.http.common.headers;
+
+import lombok.Data;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Utility class for processing our standard user agent strings.
+ *
+ * @author Petr Dvorak, petr@wultra.com
+ */
+public class UserAgent {
```

Comment:
If this should be a utility class, we may make it `final` with a _private_ constructor.